### PR TITLE
Update Readme [Future]

### PR DESCRIPTION
**Pull request by Vyoam**
(Note that this is in the Future branch)
Please review the updated README.

READMEs in SDKs of other languages would be updated in the same format once this is approved.

Thanks,
Akash Amat (aamat, visa)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,233 +42,88 @@ require 'path/to/anet_php_sdk/autoload.php';
...
+...
+
+````php
+$request = new AnetAPI\CreateTransactionRequest();
+$request->setMerchantAuthentication($merchantAuthentication);
+````
+...
 
 ## Usage Examples
+*Users of previous SDK model please note: The SDK is moving towards using a new model. This model allows us to maintain SDKs better by being more responsive to API changes. (To determine if a code is using the old model or new, a prominent difference is the usage of controllers, which is used only with the new model.) Refer the [old README](README_OLD.md) for samples of the old model.*
```
Comment (brianmc) [inline README.md]: I don't think we need to refer to old model in the updated readme.  We should keep the old usage examples but just put a readme in the top of the old model folders and clearly say this is deprecated funtionality.

File: README_OLD.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
+Authorize.Net PHP SDK
+======================
+
```
Comment (brianmc) [inline README_OLD.md]: I don't think we should have a README_OLD, rather a DeprecatedSamples.md perhaps.  It should not have the old readme but just the old usage examples with a small explanation

**Comment by brianmc** [comment]
Thank-you for the updates, they look great just a couple of small comments.

**Comment by Vyoam** [comment]
Hi @brianmc 

I've done the changes you advised:
1. remove references to old model from /README.md
2. trim /README_OLD.md  and move-rename to /lib/DeprecatedSamples.md
 
Please give this a look. Also please confirm if (/lib/) is the right place to keep the old samples.
This folder does contain all the old model classes.

Thanks,
 Akash Amat (aamat, visa)

**Comment by brianmc** [comment]
Thanks Akash, I would still like to see a very simple example in the usage section (e.g. the very simplest charge credit card with only the required fields) but also reference the samples repo.

Thanks,

Brian

**Pull request merged**